Project-iOS/XLProjectName/XLProjectName/Helpers/Extensions/UIImageView.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  UIImageView.swift
+//  XLProjectName
+//
+//  Created by XLAuthorName ( XLAuthorWebsite )
+//  Copyright © 2016 'XLOrganizationName'. All rights reserved.
+//
+
+import Foundation
+import Alamofire
+import AlamofireImage
+
+
+extension UIImageView {
+    public func setImageWithURL(url: String, filter: ImageFilter? = nil, placeholder: UIImage? = nil, completion: (Response<UIImage, NSError> -> Void)? = nil) {
```

Comment:
Definitely this should be moved to another project because is not something everyone wants to use.